bubble_sort.py

```diff
@@ -33,4 +33,19 @@ def bubble_sort(lista_numeros: list[int]):
     # TODO implemente o código fazendo ordenação baseado na ideia do bubble sort.
     # TODO troque a linha abaixo pelo return com a variável que guarda a lista como um todo.
     # TODO quando terminar de implementar apague esses comentários.
+    
+    
+    for f in range(len(lista)), 0, -1:
+      exchanging = False
+
+      for c in range(0, f -1):
```

Comment:
Isso é o mesmo que:
```suggestion
      for c in range(f -1):
```
Que é mais simples.